### ISSUE TITLE
Rule proposal: `no-blob-to-file`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1270]
Many people don't know [`File` implements `Blob`](https://developer.mozilla.org/en-US/docs/Web/API/File#instance_methods), the only difference is `File` has `.name` `..lastModified` in most cases don't need convert to `File`.

## Fail

```js
const file = new File([blob], {name: '1.jpg'});
formData.append('file', file);
```

## Pass

```js
formData.append('file', blob, '1.jpg');
```

This rule might need type info.

**Comment by sindresorhus** [comment]
This is now accepted.
